app/controllers/teams/users_controller.rb

```diff
@@ -1,5 +1,29 @@
 class Teams::UsersController < AuthenticatedController
-   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+    before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def new
+      @user = User.new
+    end
+  
+    def create
+      add_email = user_params.require(:email).downcase
+      @user = User.where(email: add_email).first || User.new(email: add_email)
+      if team.users.include?(@user)
+        @user = User.new
+        flash[:error] = I18n.t('teams.users.create.already_member', email: add_email)
+        render :new
+      elsif not @user.valid?
+        flash[:error] = @user.errors.of_kind?(:email, :invalid) ?
+                        I18n.t('teams.users.create.invalid_email', email: add_email) : 
+                        @user.errors.objects.first.full_message
+        render :new
+      elsif team.update(users: team.users + [@user])
+        flash[:success] = I18n.t('teams.users.create.success', email: add_email)
+        redirect_to team_path(team.id)
```

Comment:
`redirect_to team_path(team.id) and return` to allow you to not need to repeat the `render :new`